components.json

```diff
@@ -883,6 +883,12 @@
 			"require": "markup-templating",
 			"owner": "Golmote"
 		},
+		"sparql": {
+			"title": "SPARQL",
+			"require": "turtle",
+			"owner": "Triply",
```

Comment:
Shouldn't this be your GitHub handle?

Reply:
I've updated the handle to match the company handle we use on github. It would be me most of the time doing bugfixes, but we are better reachable this way, we thought.

Reply:
Ah, ok. I was just a little confused because usually, it's the author itself, but that also ok.